packages/libglib/libglib.spec

```
@@ -0,0 +1,66 @@
+Name: %{_cross_os}libglib
+Version: 2.68.1
+Release: 1%{?dist}
+Summary: The GLib libraries
+License: LGPL-2.1-or-later
+URL: https://www.gtk.org/
+Source0: https://download.gnome.org/sources/glib/2.68/glib-%{version}.tar.xz
+BuildRequires: meson
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libffi-devel
+BuildRequires: %{_cross_os}libmount-devel
+BuildRequires: %{_cross_os}libpcre-devel
+BuildRequires: %{_cross_os}libselinux-devel
+BuildRequires: %{_cross_os}libz-devel
+Requires: %{_cross_os}libffi
+Requires: %{_cross_os}libmount
+Requires: %{_cross_os}libpcre
+Requires: %{_cross_os}libselinux
+Requires: %{_cross_os}libz
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the GLib libraries
+Requires: %{name}
+Requires: %{_cross_os}libffi-devel
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n glib-%{version} -p1
+
+%build
+CONFIGURE_OPTS=(
```

I wonder if it'd be worth having `-Dselinux=true` (rather than the default of auto) so it'd fail if we change something about our SELinux setup and don't realize the impact on glib2, rather than having it build without support.  Maybe also `-Dlibmount=<whether you want support or not, I don't know>`